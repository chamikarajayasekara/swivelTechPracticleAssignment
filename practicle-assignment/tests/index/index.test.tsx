import {render, renderHook, screen} from '@testing-library/react'
import Home from '../../pages/index'


describe('Home', () => {
    it('renders a description', () => {
        const { container } = render(<Home />)
        const textToFind = "Welcome to Employee Management System"
        const a = screen.getByText(textToFind)
        expect(a).toBeInTheDocument()

        // expect(container).toMatchSnapshot()
    })
})