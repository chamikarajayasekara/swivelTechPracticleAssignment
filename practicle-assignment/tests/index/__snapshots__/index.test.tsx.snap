// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a description 1`] = `
<div>
  <div
    class="container-fluid"
  >
    Welcome to Employee Management System
  </div>
</div>
`;

exports[`Home renders a heading 1`] = `
<div>
  <div
    class="container-fluid"
  >
    Welcome to Employee Management System
  </div>
</div>
`;
